staging/src/kubevirt.io/api/instancetype/v1alpha2/conversion.go

```diff
@@ -42,6 +44,15 @@ func Convert_v1alpha2_CPUPreferences_To_v1beta1_CPUPreferences(in *CPUPreference
 	return autoConvert_v1alpha2_CPUPreferences_To_v1beta1_CPUPreferences(in, out, s)
 }
 
+// Manually defined function to convert to deprecated fields
+func Convert_v1alpha2_FirmwarePreferences_To_v1beta1_FirmwarePreferences(in *FirmwarePreferences, out *v1beta1.FirmwarePreferences, s conversion.Scope) error {
+	out.PreferredUseBios = (*bool)(unsafe.Pointer(in.PreferredUseBios))
+	out.PreferredUseBiosSerial = (*bool)(unsafe.Pointer(in.PreferredUseBiosSerial))
+	out.DeprecatedPreferredUseEfi = (*bool)(unsafe.Pointer(in.PreferredUseEfi))
+	out.DeprecatedPreferredUseSecureBoot = (*bool)(unsafe.Pointer(in.PreferredUseSecureBoot))
```

Comment:
Shouldn't we clone the values here? Why do we need `unsafe`? Thanks

Reply:
This is ultimately cloning these values, it's a direct copy of the original generated conversion code that's only required given the naming changes.